Altis_Life.Altis/stringtable.xml

```diff
@@ -901,6 +901,30 @@
             <German>Admin %1 hat die Debug-Konsole geöffnet.</German>
         </Key>
     </Package>
+    <Package name="Jail_Strings">
+        <Key ID="STR_JAIL_NOTIFYGLITCH">
+            <Original>You've magically glitched out of jail. This has been logged and more attempts may result in punishment.</Original>
+            <Czech>Vy jste magicky Odvázaný z vězení. Tato skutečnost byla zaznamenána a další pokusy může mít za následek trest.</Czech>
+            <Spanish>Usted ha glitched mágicamente fuera de la cárcel. Esto ha sido registrada y más intentos puede resultar en un castigo.</Spanish>
+            <Russian>Вы волшебным образом сглаженным из тюрьмы. Это было зарегистрировано и больше попыток может привести к наказанию.</Russian>
+            <German>Du hast dich magisch aus dem Gefängnis befreit. Dies wurde protokolliert und weitere Versuche könnten Bestrafungen zur Folge haben.</German>
+            <French>Vous avez par magie glitched sortir de prison. Cela a été noté et plus de tentatives peut entraîner une peine.</French>
+            <Italian>Hai magicamente glitched fuori di prigione. Questo è stato registrato e più tentativi può causare una punizione.</Italian>
+            <Portuguese>Você magicamente glitched fora da cadeia. Este foi registado e mais tentativas pode resultar em punição.</Portuguese>
+            <Polish>Masz magicznie glitched z więzienia. Zostało rejestrowane i więcej prób może skutkować karą.</Polish>
+        </Key>
+        <Key ID="STR_JAIL_NOTIFYANTIJUMP">
+            <Original>You cannot jump since you are wearing an anti-jump suit.</Original>
+            <Czech>Nemůžete skákat, protože jste na sobě anti-skok oblek.</Czech>
+            <Spanish>No se puede saltar desde que usted está usando un traje anti-salto.</Spanish>
+            <Russian>Вы не можете прыгать, так как вы носите анти-полукомбинезона.</Russian>
+            <German>Sie können nicht springen, da Sie ein Anti-Sprung-Anzug tragen.</German>
```

Comment:
Your wish is my command.

`<German>Du kannst nicht springen, da du einen Anti-Sprung-Anzug trägst.</German>`